pkgs/applications/networking/remote/freerdp/default.nix

```diff
@@ -47,6 +52,10 @@ let
       dir = "libfreerdp/crypto/test";
       file = "Test_x509_cert_info.c";
     }
+    {
```

Comment:
Could we disable this only on darwin if it's only an issue there?

Reply:
Done.